lua/octo/init.lua

```diff
@@ -149,6 +149,7 @@ function M.create_buffer(type, obj, repo, create)
   end
 
   api.nvim_set_current_buf(bufnr)
+  api.nvim_set_hl_ns(constants.OCTO_HIGHLIGHT_NS)
```

Comment:
The highlights in the issue title (line 1) dissapeared. May it be related with this?

Reply:
This is required because I set the generic hightlight groups for the bubbles with the Lua API. Such highlight groups take only effect in their namespace. As you had this namespace already declared, I was just using them. 
I could not set the highlight without the API, as I get the color of the bubbles body highlight group via the API too. And these values are confusing. Maybe they are HEX into RGB (as the function parameter suggests) and then all decimal numbers added or concatenated, but it is of type number, ... The docs don't say much about it. Passing them back into the API to set a color works (as I do).
I failed to get the color of a highlight group with Vim script commands. At least I fail to save the response of a VimL command into a lua variable. `local foo = vim.cmd("echo 'bar'")` does not work...
This is also the reason I do not use your `create_highlight` function, as of the incompatible color value schemes. I added a comment there to investigate it one day and fix. But maybe you can help?

Reply:
> This is required because I set the generic hightlight groups for the bubbles with the Lua API. Such highlight groups take only effect in their namespace. As you had this namespace already declared, I was just using them. 

Its ok, I was just trying to understand how `nvim_set_hl` works. I was and still not 100% sure if these HL groups are activated at the buffer, window or vim level. But it seems they are applied at the vim level? also not sure what happens if the activated NS does not define some HL groups, may that be the reason why Title virtual text is no longer highlighted?

> I could not set the highlight without the API, as I get the color of the bubbles body highlight group via the API too. And these values are confusing. Maybe they are HEX into RGB (as the function parameter suggests) and then all decimal numbers added or concatenated, but it is of type number, ... The docs don't say much about it. Passing them back into the API to set a color works (as I do).

They appear to be the int version of the RGB hex codes. So for example 255 is blue (0x0000FF)

> I failed to get the color of a highlight group with Vim script commands. At least I fail to save the response of a VimL command into a lua variable. `local foo = vim.cmd("echo 'bar'")` does not work...

What function were you trying to call? You should be able to do `local foo = vim.fn.some_function()`

> This is also the reason I do not use your `create_highlight` function, as of the incompatible color value schemes. I added a comment there to investigate it one day and fix. But maybe you can help?

Im ok using the new `nvim_set_hl` API, but lets figure out why we lost the title virtual text highlights

Reply:
> Im ok using the new nvim_set_hl API, but lets figure out why we lost the title virtual text highlights

I still dont know what part of this PR causes it, but the reason that Im losing the highlights in the Title line is that sign linehl has now more priority than the other hl groups. So we are also losing markdown highlighting because it gets clobbered with the sign linehl (`OctoNvimEditable`)

Reply:
> I solved this issue at the moment with using a highlight for `OctoNvimEditable` that has only a background

Worked for me! Perhaps we should add a note in the README for this HL group
Other than that, this PR looks great! thank you so much! I think we can rebase and merge :tada: